src/pages/api/graphql.page.ts

```diff
@@ -2,9 +2,9 @@ import { ApolloServer } from "apollo-server-micro";
 import Cors from "micro-cors";
 const cors = Cors();
 
-import { typeDefs } from "../../services/Apollo/typedefs";
 import neo4j from "neo4j-driver";
 import { Neo4jGraphQL } from "@neo4j/graphql";
+import { typeDefs } from "@chainverse/os";
```

Comment:
Very cool that you were able to make the type definitions as an import separate from the frontend code. I followed along to find this import coming from the `chainverse-os` repository (https://github.com/DiamondDAO/chainverse-os/search?q=typedefs).

With this import pattern I imagine we can create something like a "Data Model Marketplace" with different custom types depending on the use case of the application being built; all the developer has to do is import the data model (i.e. type def) for their specific application.